src/client/components/NavbarDropdown.css

```diff
@@ -0,0 +1,51 @@
+/* The dropdown container */
+.dropdown {
+  float: left;
+  overflow: hidden;
+
+  a {
```

Comment:
any reason the a tag styling is duplicated in .dropdown as well as .dropdownContent?

Also, intuition and some quick playing around suggest some of these attributes are unnecessary, but trust you if you think they are